src/sage/monoids/free_monoid_element.py

```diff
@@ -159,9 +159,9 @@ def _latex_(self):
             g = x[int(v[i][0])]
             e = v[i][1]
             if e == 1:
-                s += "%s " % (g,)
+                s += f"{g} "
             else:
-                s += "%s^{%s}" % (g, e)
+                s += f"{g}^{{{e}}}"
```

Comment:
f-strings for latex get weird pretty quickly. maybe we need a general helper function in `sage.typeset` for putting things into braces?